extensions/amp-story-360/0.1/amp-story-360.css

```
@@ -15,7 +15,7 @@
  * limitations under the License.
  */
 
-amp-story-360 canvas {
+ amp-story-360 canvas {
```

Nittiest nit I ever wrote:

```suggestion
amp-story-360 canvas {
```